far2l/far2sdk/farplug-wide.h

```diff
@@ -2230,6 +2235,7 @@ struct PluginStartupInfo
 	FARAPIADVCONTROL       AdvControl;
 	FARAPIADVCONTROL       AdvControlAsync;
 	FARAPIINPUTBOX         InputBox;
+	FARAPICOLORDIALOG      ColorDialog;
```

Comment:
better move it to the end so at least be back compatible whenever possible

Reply:
suggested change implemented